packages/hash/integration/src/tests/model/ontology/property-type.model.test.ts

```diff
@@ -25,22 +26,23 @@ const graphApi = createGraphClient(logger, {
   port: graphApiPort,
 });
 
-let ownedById: string;
+let testUser: UserModel;
+let testUser2: UserModel;
 let textDataTypeModel: DataTypeModel;
 let propertyTypeSchema: Omit<PropertyType, "$id">;
 
 beforeAll(async () => {
-  const testUser = await createTestUser(graphApi, "property-type-test", logger);
-
-  ownedById = testUser.entityId;
+  testUser = await createTestUser(graphApi, "property-type-test", logger);
+  testUser2 = await createTestUser(graphApi, "property-type-test", logger);
```

Comment:
See comment about same string and conflicts

Reply:
See https://github.com/hashintel/hash/pull/1197#discussion_r999293315